addons/sourcemod/scripting/MakoVote.sp

```diff
@@ -42,6 +40,9 @@ public void OnPluginStart()
 	g_cDelay = CreateConVar("sm_makovote_delay", "3.0", "Time in seconds for firing the vote from admin command", FCVAR_NOTIFY, true, 1.0, true, 10.0);
 	g_cRtd = CreateConVar("sm_makovote_rtd", "0", "Enable Roll The Dice", FCVAR_NOTIFY, true, 0.0, true, 1.0);
 	g_cRtd_Percent = CreateConVar("sm_makovote_rtd_percent", "15", "Percentage chance value to trigger ZM mod with RTD", FCVAR_NOTIFY, true, 0.0, true, 100.0);
+	g_cMenuCDWhiteDraw = CreateConVar("sm_makovote_menu_cdwhitedraw", "1", "Enable/Disable the white draw of stage when it is on cooldown", FCVAR_NOTIFY, true, 0.0, true, 1.0);
+	g_cZMStageMenu = CreateConVar("sm_makovote_zmstage_menu", "1", "Enable/Disable the ZM stage in the menu [dependency: sm_makovote_rtd 0]", FCVAR_NOTIFY, true, 0.0, true, 1.0);
+	g_cCDNumber = CreateConVar("sm_makovote_cd_maxstages", "3", "Number of stages to be on cooldown before reset", FCVAR_NOTIFY, true, 0.0, true, NUMBEROFSTAGES.0);
```

Comment:
wtf `NUMBEROFSTAGES.0` 

instead do float(NUMBEROFSTAGES)

Reply:
My bad I was curious to see if that work and I forget to remove it 😂